qemu/tests/numa_prealloc_handling.py

```diff
@@ -0,0 +1,26 @@
+import re
+from avocado.utils import process
+
+
+def run(test, params, env):
+    """
+    numa_prealloc_threads test
+    1) Test
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment
+    """
+    cmd_suffix = params.get("cmd_suffix")
+    cmd_line = params.get("cmd_fixed") + cmd_suffix
+    cmd_line_tc = params.get("qemu_cmd") + cmd_suffix
```

Comment:
For this part, we could use the cartesian to implement, like:
```suggestion
    cmd_without_tc = params.get("cmd_without_prealloc")
    cmd_with_tc = params.get("cmd_with_prealloc") 
```
And we could update the cfg, like:
```suggestion
    cmd_time_taskset = xxxx
    cmd_qemu_binary_path = xxx
    cmd_qemu_options = xxxx
    cmd_option_tc = xxxx
    cmd_without_tc = "${cmd_time_taskset} ${cmd_qemu_binary_path} ${cmd_qemu_options} ; } 2>&1 | grep real"
     cmd_with_tc = "${cmd_time_taskset} ${cmd_qemu_binary_path} ${cmd_qemu_options},cmd_option_prealloc ; } 2>&1 | grep real"
```
I think that it will be more flexible and configurable for upstream and downstream configurations.
Please let me know your opinions, thanks.

Reply:
Yeah that's a good consideration, code updated, thanks !